samples/snippets/quickstart.py

```diff
@@ -0,0 +1,246 @@
+        iterable collections of Instance objects as values.
+    """
+    instance_client = gce.InstancesClient()
+    agg_list = instance_client.aggregated_list(project=project)
+    all_instances = {}
+    for zone, response in agg_list:
+        if response.instances:
+            all_instances[zone] = response.instances
+    return all_instances
+
+
+# [END compute_instances_list_all]
+
+
+# [START compute_instances_create]
+def create_instance(
+    project: str, zone: str, machine_type: str, machine_name: str, source_image: str
+) -> gce.Instance:
+    """
+    Sends an instance creation request to GCP and waits for it to complete.
+
+    Args:
+        project: Name of the project you want to use.
+        zone: Name of the zone you want to use, for example: us-west3-b
+        machine_type: Machine type you want to create in following format:
+            "zones/{zone}/machineTypes/{type_name}". For example:
+            "zones/europe-west3-c/machineTypes/f1-micro"
+        machine_name: Name of the new machine.
+        source_image: Path the the disk image you want to use for your boot
+            disk. This can be one of the public images
+            (e.g. "projects/debian-cloud/global/images/family/debian-10")
+            or a private image you have access to.
+
+    Returns:
+        Instance object.
+    """
+    instance_client = gce.InstancesClient()
+
+    # Every machine requires at least one persistent disk
+    disk = gce.AttachedDisk()
+    initialize_params = gce.AttachedDiskInitializeParams()
+    initialize_params.source_image = (
+        source_image  # "projects/debian-cloud/global/images/family/debian-10"
+    )
+    initialize_params.disk_size_gb = "10"
+    disk.initialize_params = initialize_params
+    disk.auto_delete = True
+    disk.boot = True
+    disk.type_ = gce.AttachedDisk.Type.PERSISTENT
+
+    # Every machine needs to be connected to a VPC network.
+    # The 'default' network is created automatically in every project.
+    network_interface = gce.NetworkInterface()
+    network_interface.name = "default"
+
+    # Collecting all the information into the Instance object
+    instance = gce.Instance()
+    instance.name = machine_name
+    instance.disks = [disk]
+    instance.machine_type = (
+        machine_type  # "zones/europe-central2-a/machineTypes/n1-standard-8"
+    )
+    instance.network_interfaces = [network_interface]
+
+    # Preparing the InsertInstanceRequest
+    request = gce.InsertInstanceRequest()
+    request.zone = zone  # "europe-central2-a"
+    request.project = project  # "diregapic-mestiv"
+    request.instance_resource = instance
+
+    print(f"Creating the {machine_name} instance in {zone}...")
+    operation = instance_client.insert(request=request)
+    # wait_result = operation_client.wait(operation=operation.name, zone=zone, project=project)
+    operation = wait_for_operation(operation, project)
```

Comment:
The LRO experience looks different from existing GAPICs. @vam-google Does the Compute API have a non-standard LRO representation? It doesn't look like the generator produced the usual `google.api_core.Operation` type.

For comparison, see videointelligence where you can do `operation.result(timeout=90)` to poll for the result.

- [sample](https://github.com/googleapis/python-videointelligence/blob/da555fe9727942ef39746356095cd0292ea6472e/samples/analyze/analyze.py#L46-L51)
- [method documentation](https://googleapis.dev/python/videointelligence/latest/videointelligence_v1/video_intelligence_service.html#google.cloud.videointelligence_v1.services.video_intelligence_service.VideoIntelligenceServiceClient.annotate_video)
- [google.api_core.operation.Operation](https://googleapis.dev/python/google-api-core/latest/operation.html)

Reply:
@busunkim96 would a simpler approach suggest by @bshaffer work here?

Please refer to https://github.com/GoogleCloudPlatform/php-docs-samples/blob/80801c983457f407cb9a81d3b60654369f090e03/compute/cloud-client/instances/src/create_instance.php#L80

Reply:
This is how I saw the team use the operations in their test code. It is an instance of `google.cloud.compute_v1.types.Operation` and not `google.api_core.Operation`.